themes/bootstrap3/less/components/sidebar.less

```diff
@@ -204,3 +224,18 @@ h3.facet-title {
   .author-facets { display: flex; }
   .author-list { flex: 0 1 100%; }
 }
+
+.multi-filters-selection {
+  position: sticky;
+  top: 0;
```

Comment:
Does this make the JS I commented on earlier redundant?

Reply:
Yes